### PR TITLE
Add OLMv0 downstream component owners config to prow

### PR DESCRIPTION
**Pull request by tmshort**

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -2806,6 +2806,9 @@ owners:
     openshift/operator-framework-catalogd:
       owners: DOWNSTREAM_OWNERS
       owners_aliases: DOWNSTREAM_OWNERS_ALIASES
+    openshift/operator-framework-olm:
+      owners: DOWNSTREAM_OWNERS
+      owners_aliases: DOWNSTREAM_OWNERS_ALIASES
```
Comment (dtfranz) [inline core-services/prow/02_config/_plugins.yaml:2811]: I don't see the `DOWNSTREAM_OWNERS_ALIASES` file in the repo yet, and I also don't see it in your downstream OLM PR. Do you have another PR to add this file somewhere?

Comment (tmshort) [reply core-services/prow/02_config/_plugins.yaml:2811]: It's a "just in case we add it" thing.

Comment (dtfranz) [reply core-services/prow/02_config/_plugins.yaml:2811]: Ah OK. As long as the CI is happy with it I am :)

**Comment by tmshort** [comment]
ping @dptp ?

**Comment by tmshort** [comment]
ping @bear-redhat @jmguzik ?
